Doc/library/unittest.mock.rst

```diff
@@ -702,6 +702,19 @@ the *new_callable* argument to :func:`patch`.
         unpacked as tuples to get at the individual arguments. See
         :ref:`calls as tuples <calls-as-tuples>`.
 
+        .. warning::
+
+            The way :attr:`mock_calls` are recorded means that where nested
+            calls are made, the parameters of ancestor calls are not recorded
```

Comment:
They are recorded, just as different calls.. :/ I think this is going to be really confusing for the reader.

Reply:
The parameters are absolutely not recorded in `mock_calls`, this is the core problem that isn't solvable, as I explained.

Reply:
They are! just not in the last one, look:

```
>>> mock_backend.mock_calls
[call.get_endpoint('foobar'),
 call.get_endpoint().create_call('spam', 'eggs'),
 call.get_endpoint().create_call().start_call()]
```

Reply:
I'm not sure we're getting anywhere constructive with this, so let's wait for Michael to take a look.

Reply:
...except when `get_endpoint` is called more than once.